source/includes/_api.auth.md

```diff
@@ -37,3 +39,114 @@ var vhxjs = new vhx('YOUR_API_KEY');
+
+> <h5 class="is-internal head-5 text--white">OAuth with Pairing Codes</h5>
+
+> <span class="is-internal">Example Fetch Code Request</span>
+
+```plaintextinternal
+curl -X POST "https://api.vhx.tv/oauth/codes"
+     -d client_id={client_id}
+     -d client_secret={client_secret}
+```
+
+> <span class="is-internal">Example Response</span>
+
+```jsoninternal
+{
+  "code": "YOUR_PAIRING_CODE"
+}
+```
+
+> <span class="is-internal">Example Polling Request</span>
+
+```plaintextinternal
+curl -G "https://api.vhx.tv/oauth/codes/{code}" \
+     -d client_id={your_client_id} \
+     -d client_secret={your_client_secret}
+```
+
+> <span class="is-internal">Example Response</span>
+
+```jsoninternal
+{
+  "access_token": "YOUR_ACCESS_TOKEN",
+  "token_type": "bearer",
+  "expires_in": 7200,
+  "refresh_token": "YOUR_REFRESH_TOKEN",
+  "scope": "read"
+}
+```
+
+> <span class="is-internal">Example Refresh Token Request</span>
+
+```plaintextinternal
+$ curl -X POST "https://api.vhx.tv/oauth/token" \
+  -d grant_type=refresh_token \
+  -d refresh_token={your_refresh_token}
+```
+> <span class="is-internal">Example Response</span>
+
+```jsoninternal
+{
+  "access_token": "YOUR_NEW_ACCESS_TOKEN",
+  "token_type": "bearer",
+  "expires_in": 7200,
+  "refresh_token": "YOUR_NEW_REFRESH_TOKEN",
+  "scope": "public"
+}
+```
+
+<section class="is-internal text-2 contain">
+  <p>To make requests on behalf of a user for accessing content available to them, you will need to authorize the user via application <strong>Pairing Codes</strong>.</p>
+
+  <h4><strong>1. Fetch the code</strong></h4>
+  <p>Applications should request a code from our API, display it on the screen for the customer, and then direct them to <code>https://{subdomain}.vhx.tv/activate</code> to enter the code.</p>
+
+  <h4><strong>2. Poll for user activation</strong></h4>
+  <p>Upon displaying the code to the customer, you should begin polling <code>/oauth/codes/{code}</code> for the code at a ~2.5 second interval with a max of 500 retries.</p>
+  <p>This will return a <code>404</code> until the customer correctly pairs it with their account. At that point a <code>200</code> will be returned with an <code>access_token</code>, <code>refresh_token</code>, and <code>expiry</code> information. This information should be saved in your device's registry and is what deems the device as being connected/linked to a logged in customer.</p>
+  <p>You can than make requests on behalf of the customer using the Authorization Bearer header.</p>
+
+  <h4><strong>3. Refresh Tokens</strong></h4>
+  <p>Access tokens for a user expire after 2 hours. To continue making API requests you can refresh your access token by requesting a new one (from the <code>/oauth/token</code> endpoint) using the provided <code>refresh_token</code> in the previous response.</p>
+</section>
```

Comment:
there are lot's of references to "user" here. let's maybe try and stick with "customer" ?